Makefile

```diff
@@ -44,10 +44,13 @@ python:
 	venv/bin/python -m pip install -r requirements.txt
 
 run: python
-	PATH=./venv/bin:$(PATH) RUST_LOG=debug RUST_BACKTRACE=full cargo run -- --config config/local.toml
+	PATH="./venv/bin:$(PATH)" RUST_LOG=debug RUST_BACKTRACE=full cargo run -- --config config/local.toml
```

Comment:
This fixes an unrelated bug that causes `make run` to fail if an item in your path has any whitespace characters in it.